server/src/Lib.hs

```diff
@@ -32,7 +32,7 @@ import qualified System.Process as Proc
 --   localhost:8081/procps/ps?sys=aarch64-linux
 --  TODO API documentation
 type API =
-  Get '[JSON] NoContent
+  Get '[PlainText] NoContent
```

Comment:
I don't think it really matters what this content-type is set to.

What I really wanted to have is:

```haskell
  Get '[] Void
```

Since the redirect is performed with `throwError`, it makes it more obvious that the handler for this doesn't actually return anything.

But servant doesn't appear to be able to handle an empty content-type list.